src/domains/portfolio/components/ImportWallet/ImportDetailStep.tsx

```diff
@@ -64,14 +64,14 @@ const MnemonicField = ({
 					required: t("COMMON.VALIDATION.FIELD_REQUIRED", {
 						field: label,
 					}).toString(),
-					validate: (value) =>
-						validateAddress({
-							findAddress,
-							network,
-							profile,
-							t,
-							value,
-						}),
+					validate: async (value: string) => {
+						const count = value.trim().split(/\s+/).length;
+						if (count !== 12 && count !== 24) {
+							return t("COMMON.INPUT_ADDRESS.VALIDATION.NOT_VALID");
```

Comment:
"address is not valid" isn't true for the message either, it's the mnemonic that's invalid